frontend/src/app/admin/components/admin-content/admin-autorites/admin-autorites.component.html

```diff
@@ -0,0 +1,150 @@
+              <tbody>
+              <tr *ngFor="let autorite of autorites">
+                <td>
+                  <input type="checkbox" (click)="assignId(autorite.id)" [checked]="allChecked">
+                </td>
+                <td>{{ autorite.nom }}</td>
+                <td>{{ autorite.prenom }}</td>
+                <td>{{ autorite.poste }}</td>
+                <td><a href="#" (click)="editForm(autorite.id)" data-toggle="modal" data-target="#modal-autorite-form"><i
+                  class="fa fa-edit"></i></a></td>
+              </tr>
+              </tbody>
+            </table>
+          </div>
+          <!-- /.box-body -->
+        </div>
+        <!-- /.box -->
+        <div class="col-md-3 {{requestMessage.visibility}}">
+          <div class="box box-{{requestMessage.type}} box-solid">
+            <div class="box-header with-border">
+              <h3 class="box-title">{{requestMessage.title}}</h3>
+
+              <div class="box-tools pull-right">
+                <button type="button" class="btn btn-box-tool" data-widget="remove"><i class="fa fa-times"></i></button>
+              </div>
+              <!-- /.box-tools -->
+            </div>
+            <!-- /.box-header -->
+            <div class="box-body">
+              {{requestMessage.message}}
+            </div>
+            <!-- /.box-body -->
+          </div>
+          <!-- /.box -->
+        </div>
+      </div>
+      <div class="col-xs-3 {{requestMessage.visibility}}">
+        <div class="box box-{{requestMessage.type}} box-solid">
+          <div class="box-header with-border">
+            <h3 class="box-title">{{requestMessage.title}}</h3>
+
+            <div class="box-tools pull-right">
+              <button type="button" class="btn btn-box-tool" data-widget="remove"><i class="fa fa-times"></i></button>
+            </div>
+            <!-- /.box-tools -->
+          </div>
+          <!-- /.box-header -->
+          <div class="box-body">
+            {{requestMessage.message}}
+          </div>
+          <!-- /.box-body -->
+        </div>
+        <!-- /.box -->
+      </div>
+    </div>
+
+    <!-- Modal edit -->
+    <div class="modal fade" id="modal-autorite-form" style="display: none;">
+      <div class="modal-dialog">
+        <div class="modal-content">
+          <div class="modal-header">
+            <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+              <span aria-hidden="true">×</span></button>
+            <h4 class="modal-title">{{formTitle}}</h4>
+          </div>
+          <div class="modal-body">
+            <form (ngSubmit)="submit(autorite)" [formGroup]="autoriteForm">
+              <div class="box-body">
+                <div class="form-group">
+                  <label>Nom*</label>
+                  <input type="text" class="form-control" formControlName="nom" placeholder="Entrer le nom" data-toggle="tooltip" data-placement="top" title="Caracteres majuscules et minuscules">
```

Comment:
The title does not match the requirement. Please, remove it since the start already mention that this field is required.

This comment applies for the lines 127 and 131